### ISSUE TITLE
Compilation error with ES6 strings

### ISSUE THREAD

**Issue by hakatashi** [pugjs/jade#2198]
```jade
a(href=`${42}/${42}`)
```

produces following error:

```
Jade:1
  > 1| a(href=`${42}/${42}`)

The end of the string was reached with no closing bracket found.
```

Tested with http://jade-lang.com/demo/

**Comment by TimothyGu** [comment]
This code gives a different error in master, but will be fixed by pugjs/pug-lexer#53.

**Comment by hakatashi** [comment]
Thanks! :sunglasses:
